sdk/java/src/main/java/io/juicefs/JuiceFileSystemImpl.java

```diff
@@ -472,7 +472,7 @@ private String readFile(String file) {
               (name != null && name.equals(uri.getAuthority()))) {
         fs = this;
       } else {
-        fs = path.getFileSystem(getConf());
```

Comment:
Is it too slow to create a filesystem to open single file?

If the fs is closed, can we capture the exception and try again?